library/core/src/error.rs

```diff
@@ -130,6 +130,7 @@ pub trait Error: Debug + Display {
     ///
     /// ```rust
     /// #![feature(error_generic_member_access)]
+    /// #![feature(error_in_core)]
```

Comment:
Would it make more sense to change the import to `std::error::{...}`? If we don't do that then stabilization of the `error_generic_member_access` feature would be unnecessarily dependent on `error_in_core`, right?